packages/client/components/JiraScopingSearchResultsRoot.tsx

```diff
@@ -1,55 +1,38 @@
 import graphql from 'babel-plugin-relay/macro'
-import React from 'react'
-import {createFragmentContainer, QueryRenderer} from 'react-relay'
-import {JiraScopingSearchResultsRootQuery} from '~/__generated__/JiraScopingSearchResultsRootQuery.graphql'
-import useAtmosphere from '../hooks/useAtmosphere'
+import React, {Suspense} from 'react'
+import {createFragmentContainer} from 'react-relay'
 import {JiraScopingSearchResultsRoot_meeting} from '../__generated__/JiraScopingSearchResultsRoot_meeting.graphql'
 import ErrorComponent from './ErrorComponent/ErrorComponent'
 import JiraScopingSearchResults from './JiraScopingSearchResults'
-
-const query = graphql`
-  query JiraScopingSearchResultsRootQuery(
-    $teamId: ID!
-    $queryString: String
-    $isJQL: Boolean!
-    $projectKeyFilters: [ID!]
-    $first: Int
-  ) {
-    viewer {
-      ...JiraScopingSearchResults_viewer
-    }
-  }
-`
+import ErrorBoundary from './ErrorBoundary'
+import useQueryLoaderNow from '../hooks/useQueryLoaderNow'
+import jiraScopingSearchResultsQuery, {
+  JiraScopingSearchResultsQuery
+} from '../__generated__/JiraScopingSearchResultsQuery.graphql'
+import MockScopingList from '~/modules/meeting/components/MockScopingList'
 
 interface Props {
   meeting: JiraScopingSearchResultsRoot_meeting
 }
 
 const JiraScopingSearchResultsRoot = (props: Props) => {
-  const atmosphere = useAtmosphere()
   const {meeting} = props
   const {teamId, jiraSearchQuery} = meeting
   const {queryString, projectKeyFilters, isJQL} = jiraSearchQuery
   const normalizedQueryString = queryString.trim()
-
+  const queryRef = useQueryLoaderNow<JiraScopingSearchResultsQuery>(jiraScopingSearchResultsQuery, {
+    teamId,
+    queryString: normalizedQueryString,
+    isJQL,
+    projectKeyFilters: projectKeyFilters as string[],
+    first: 100
+  })
   return (
-    <QueryRenderer<JiraScopingSearchResultsRootQuery>
-      environment={atmosphere}
-      query={query}
-      variables={{
-        teamId,
-        queryString: normalizedQueryString,
-        isJQL,
-        projectKeyFilters: projectKeyFilters as string[],
-        first: 100
-      }}
-      fetchPolicy={'store-or-network' as any}
-      render={({props, error}) => {
-        const viewer = props?.viewer ?? null
-        if (error) return <ErrorComponent error={error} eventId={''} />
-        return <JiraScopingSearchResults viewer={viewer} meeting={meeting} />
-      }}
-    />
+    <ErrorBoundary fallback={(error) => <ErrorComponent error={error} eventId={''} />}>
```

Comment:
Not quite sure if I need to pass the `eventId` in from the `fallback()` parameter of the `ErrorBoundary`

Reply:
cool, I just saw the `render()` method of `ErrorComponent`, so by default, we don't need the error fallback.

Reply:
Yes, that's also something I learned while doing this review